data/json/snippets/radio.json

```diff
@@ -13,17 +14,31 @@
+      "It's time to line up some fresh tracks and for y’all to crank up the volume (don’t do that if you’re in an infested area), because I’m DJ Dust Bowl and you're rocking along to Dark Days Radio.  It’s just me and Sam in the studio again, folks.  Anarchie’s on the road back home, and we just wanna say that it was a pleasure having him join us in manning Dark Days Radio for a while.  So, Anarchy, if you’re driving along and tuning in as well, have a safe trip, you sheet metal-wearing lunatic.  Hallelujah to have had someone like you kick around the set, and hallelujah for all the new tracks you brought in for us!",
+      "Attention folks, this is DJ Dustbowl in with an emergency announcement for anyone that happens to be listening.  I don’t know if it’s just localized to here or is widespread, but crap’s going down outside.  Reality’s breaking, people, just like when this shitshow started.  Stay indoors at all costs, close any curtains and barricade the doors for good measure, and, for the love of God, if you hear voices outside, ignore them!  They’re not human.  We’re going to be going off air and heading below ground while this crapshoot runs its course.  Have faith, people, we’ll all get through this one as well.  Just don’t go Luny on me, and again, don’t, listen, to the voices!",
+      "Ok, ladies and gentlemen, this is DJ Dustbowl, you’re tuned into Dark Days Radio, and if you’re hearing this, then let’s have an amen because it means we all made it out of the storm.  That is, if the storm was a wide-spread thing and not just neatly centered atop my head.  If that’s the case and y’all didn’t catch my broadcast, the bottom line's that reality was breaking, just like when this nonsense started.  Now, I don’t know if it’ll happen again, but let’s use today’s survival tip slot to see what we can learn from it.  Number one, folks, if reality’s shitting the bed, don’t try to fight anything that appears.  If you’re caught out, my advice is to just run for cover, and you’d be best served by snuggling underground.  Failing that, closing all the kirtans and barricading the doors will work in a pinch.  Most importantly, people, if you hear something outside, regardless of what it might be, stuff your ears and ignore it.  You’ll hear things that will make you want to open the front door.  I heard… I opened up.  You don’t want to see what’ll be waiting for you.",
```

Comment:
```suggestion
      "Ok, ladies and gentlemen, this is DJ Dustbowl, you’re tuned into Dark Days Radio, and if you’re hearing this, then let’s have an amen because it means we all made it out of the storm.  That is, if the storm was a widespread thing and not just neatly centered atop my head.  If that’s the case and y’all didn’t catch my broadcast, the bottom line's that reality was breaking, just like when this nonsense started.  Now, I don’t know if it’ll happen again, but let’s use today’s survival tip slot to see what we can learn from it.  Number one, folks, if reality’s shitting the bed, don’t try to fight anything that appears.  If you’re caught outside, my advice is to just run for cover, and you’d be best served by snuggling underground.  Failing that, closing all the curtains and barricading the doors will work in a pinch.  Most importantly, people, if you hear something outside, regardless of what it might be, stuff your ears and ignore it.  You’ll hear things that will make you want to open the front door.  I heard… I opened up.  You don’t want to see what’ll be waiting for you.",
```